lib/cosmos/interfaces/dart_status_interface.rb

```diff
@@ -0,0 +1,90 @@
+# encoding: ascii-8bit
+
+# Copyright 2018 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+
+require 'cosmos/interfaces/interface'
+require 'cosmos/io/json_drb_object'
+require 'cosmos/utilities/sleeper'
+
+module Cosmos
+
+  # Defines all the attributes and methods common to all interface classes
+  # used by COSMOS.
+  class DartStatusInterface < Interface
+    # Initialize default attribute values
+    def initialize
+      super()
+      @write_raw_allowed = false
+      @dart = nil
+      @first = true
+      @query_time = nil
+    end
+
+    # Connects the interface to its target(s). Must be implemented by a
+    # subclass.
+    def connect
+      super()
+      @first = true
+      @status_packet = System.telemetry.packet('DART', 'STATUS').clone
+      @status_packet.write('PACKETID', 1)
+      @clear_errors_command = System.commands.packet('DART', 'CLEARERRORS')
```

Comment:
I didn't notice this before but I'd prefer CLEAR_ERRORS here with an underscore.